x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/logical/UnresolvedRelation.java

```diff
@@ -75,6 +78,11 @@ public boolean resolved() {
         return false;
     }
 
+    @Override
+    public String commandName() {
```

Comment:
Same here, please. A javadoc to explain what this method is for.
Also, if you change the name of the variable, the method name should be changed as well.